### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fizakhan90**
added the steps for how to run the project in the local system

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,19 @@ To be updated
...
+## :computer: How to Start Contributing <a name = "how_contribute"></a>
+Before contributing let's learn how to set up the project in the local system.
+### How to run the project in local system
+- First fork this repository from the option given at the top of this page.
+- Now clone the forked repository by running this command in the terminal:<br>
+```
+git clone https://github.com/<your-username>/Vritant.git
+```
+- Either create an android emulator or connect a physical device to run the project.
+- Now go to the Run command and choose Run without debugging or simply press Ctrl + F5.
```
Comment (bhattabhi013) [inline README.md:122]: Please also the terminal commands i.e `flutter run`

**Comment by fizakhan90** [comment]
please review the changes @bhattabhi013

**Review by bhattabhi013 — APPROVED** [review]

**Comment by bhattabhi013** [comment]
LGTM ✨

**Comment by fizakhan90** [comment]
Hey, just one last thing can you add the gssoc23 label to this?

**Pull request merged**